check.py

```diff
@@ -628,7 +641,7 @@ def main():
   if num_failures > 0:
     print '\n[ ' + str(num_failures) + ' failures! ]'
 
-  return num_failures
+  return 0 if num_failures == 0 else 1
```

Comment:
This fixes a bug where if some multiple of 256 tests fail, the exit code would be 0 (on linux, anyway).

Reply:
That should be a comment inline, because reversing this is exactly the sort of thing someone would think is a reasonable simplification

Reply:
Because I don't like python's ternary syntax (maybe I'm biased), how about moving the `return 1` into the block above as an early out and then just `return 0` at the end here?

Reply:
@jgravelle-google Good point. I'm not sure a comment makes sense anymore with the change @sbc100 suggested, though.

Reply:
yay to code changes to make comments not-needed

Reply:
Huh, yeah merging it with the num_failures clause above makes that even nicer :+1: